src/app/layouts/base/pages/components/applied-transactions-list/applied-transactions-list.component.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  CdkDragDrop,
+  copyArrayItem,
+  moveItemInArray,
+} from '@angular/cdk/drag-drop';
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  OnInit,
+} from '@angular/core';
+import { Transaction } from 'src/app/interfaces/transactions.interface';
+import { TransactionDto } from '../../../classes/transaction-dto.class';
+import { TransactionApiService } from '../../../services/transaction-api.service';
+import { AppliedTransactionsListResponse } from './applied-transactions-list.class';
+
+@Component({
+  selector: 'intr-applied-transactions-list',
+  templateUrl: './applied-transactions-list.component.html',
+  styleUrls: ['./applied-transactions-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppliedTransactionsListComponent implements OnInit {
+
+  public appliedTransactionsArray: Transaction[] = [];
+
+  public pointerIsOverDropList = false;
+
+  constructor(
+    private transactionApiService: TransactionApiService,
+    private changeDetectorRef: ChangeDetectorRef
+  ) {}
+
+  ngOnInit(): void {
+    this.refreshTransactionList();
+  }
+
+  public handleDrop = (event: CdkDragDrop<Transaction[]>): void => {
+    const { isPointerOverContainer, previousContainer, container } = event;
```

Comment:
const { isPointerOverContainer, previousContainer, container } = event; - you working with TS and you need specify type in anytime like - const { isPointerOverContainer, previousContainer, container } : YourTYpe = event;